pkg/vdbgen/vdb.go

```diff
@@ -171,6 +188,65 @@ func (d *DBGenerator) setShardCount(ctx context.Context) error {
 	return nil
 }
 
+// setKsafety will fetch the ksafety from the database and set it inside v.vdb
+func (d *DBGenerator) setKSafety(ctx context.Context) error {
+	q := Queries[KSafetyQueryKey]
+	rows, err := d.Conn.QueryContext(ctx, q)
+	if err != nil {
+		return fmt.Errorf("failed running '%s': %w", q, err)
+	}
+	defer rows.Close()
+
+	if rows.Err() != nil {
+		return fmt.Errorf("failed running '%s': %w", q, rows.Err())
+	}
+	if !rows.Next() {
+		return errors.New("could not get ksafety from meta-function GET_DESIGN_KSAFE()")
+	}
+	var designKSafe string
+	ksafety := vapi.KSafety1
+	if err := rows.Scan(&designKSafe); err != nil {
+		return fmt.Errorf("failed running '%s': %w", q, err)
+	}
+	if designKSafe == "0" {
```

Comment:
Can we use the constant for "0"?

```suggestion
	if designKSafe == string(vapi.KSafety0) {
```